docs-ref-conceptual/eclipse/azure-toolkit-for-eclipse-sign-in-instructions.md

```diff
@@ -45,6 +45,15 @@ To sign in Azure by device login, do the following:
 
    ![The Azure Login Dialog window][I03]
 
+> [!NOTE]
+>
+> If the browser doesn't open, configure eclipse to use an external browser like IE or Chrome:
```

Comment:
Please adjust `Eclipse`, and replace `IE or Chrome` with `Microsoft Edge, Firefox, or Chrome`

Reply:
Because eclipse doesn't have that option of `Microsoft Edge`, will use `Internet Explorer` instead.